third_party/2and3/mypy_extensions.pyi

```diff
@@ -1,8 +1,26 @@
-from typing import Dict, Type, TypeVar, Optional, Union, Any, Generic
+import abc
+import sys
+from typing import (
+    Dict, Type, TypeVar, Optional, Union, Any, Generic, Mapping, ItemsView, KeysView, ValuesView
+)
 
 _T = TypeVar('_T')
 _U = TypeVar('_U')
 
+# Internal mypy fallback type for all typed dicts (does not exist at runtime)
+class _TypedDict(Mapping[str, object], metaclass=abc.ABCMeta):
+    def copy(self: _T) -> _T: ...
+    def setdefault(self, k: str, default: object) -> object: ...
+    # Mypy expects that 'default' has a type variable type
+    def pop(self, k: str, default: _T = ...) -> object: ...
```

Comment:
This looks a bit tricky. Is it because of a plugin? I would add some more details in the comment above.

Reply:
Ok, I'll update the comment.